go_modules/Earthfile

```diff
@@ -0,0 +1,59 @@
+VERSION 0.6
+FROM ruby:2.7.1
+WORKDIR /dependabot-go_modules
+
+ARG GOLANG_VERSION=1.18
+
+SETUP:
+    COMMAND
+
+    ENV DEPENDABOT_NATIVE_HELPERS_PATH="${DEPENDABOT_NATIVE_HELPERS_PATH:-/opt}"
+    
+    DO ../common+CREATE_DEPENDABOT_USER
+    
+    COPY --chown=dependabot:dependabot +go/go "$DEPENDABOT_NATIVE_HELPERS_PATH/go"
+    ENV PATH="$DEPENDABOT_NATIVE_HELPERS_PATH/go/bin:$PATH"
+
+    COPY --chown=dependabot:dependabot +helpers/helper $DEPENDABOT_NATIVE_HELPERS_PATH/go_modules/bin/helper
+
+go:
+    FROM golang:$GOLANG_VERSION-buster
```

Comment:
If this was a Dockerfile we could have Dependabot keep this up to date for us.

I wonder how much effort it would be to support updating Earthfiles? Just a random thought... not something we need to work on now.

Reply:
BTW `earthly debug ast ./Earthfile | jq` returns the AST, which can be easily interpreted. The AST format is just json and is spec'd here: https://github.com/earthly/earthly/blob/main/ast/spec/earthfile.go#L47.

A `FROM` looks like this, for example:

```json
{"name":"FROM","args":["golang:1.17-alpine3.14"]}
```